api-docs/rst/dev-guide/README.md

```diff
@@ -0,0 +1,70 @@
+# Rackspace Autoscale API documentation
+
+**Build:** [Strider](http://ci.deconst.horse/) (http://ci.deconst.horse/rackerlabs/otter/)
+
+## Purpose
+
+This GitHub repository contains the source files for the following Rackspace Autoscale API documentation:
+
+* [Getting Started Guide](https://developer.rackspace.com/docs/autoscale/v1/developer-guide/#getting-started)
+* [Developer Guide](https://developer.rackspace.com/docs/autoscale/v1/developer-guide/)
+* [API Reference](https://developer.rackspace.com/docs/autoscale/v1/developer-guide/#api-reference)
+* [Release Notes](https://developer.rackspace.com/docs/autoscale/v1/developer-guide/#release-notes)
+
+
+## Contributing
+
+Contributions are welcome! 
+
+* To suggest changes or report a problem, submit an [issue](https://github.com/rackerlabs/otter/issues). 
+
+* To make changes to a project, create your own fork of the repository. Then, submit a [pull 
```

Comment:
Do you think this should be changed to `To make changes to the documentation` since changes to the project would be considered changes to the code which is documented in https://github.com/rackerlabs/otter/blob/master/CONTRIBUTING.md